python/lsst/meas/algorithms/astrometrySourceSelector.py

```diff
@@ -0,0 +1,121 @@
+# See COPYRIGHT file at the top of the source tree.
```

Comment:
Can we do this now? It's nice and short, but I don't see a COPYRIGHT file anywhere in the meas_algorithms repo.